### ISSUE TITLE
Bug: Empty sprite can break graphics rendering when using WebGPU + Chrome in 8.0.0-rc.9

### ISSUE THREAD

**Issue by SuperSodaSea** [pixijs/pixijs#10199]
### Current Behavior

Adding an empty sprite can break graphics rendering when using WebGPU + Chrome in 8.0.0-rc.9, as shown in the code in *Steps to Reproduce*.
Note that the empty sprite does not always break the rendering, changing the content of the graphic may give a different result.

When using WebGPU:

![Wrong](https://github.com/pixijs/pixijs/assets/8724868/1c3d2df4-2da7-4de1-bc58-68d073c3d67e)

### Expected Behavior

The result when using WebGPU should match the result with WebGL.

When using WebGL:

![Correct](https://github.com/pixijs/pixijs/assets/8724868/92ee7a6a-7a5f-4882-8ea4-74777c76c603)

### Steps to Reproduce

```js
(async () => {

const app = new PIXI.Application();
await app.init({
    preference: 'webgpu', // 'webgl'
    hello: true,
    width: 300,
    height: 300,
    backgroundColor: 'black',
});
document.body.appendChild(app.canvas);

const createGraphics = () => {
    return new PIXI.Graphics()
        .circle(0, 0, 50)
        .fill('blue')
        .stroke({ width: 10, color: 'red', alignment: 0 });
}

const graphics1 = createGraphics();
graphics1.position.set(100, 100);
app.stage.addChild(graphics1);

app.stage.addChild(new PIXI.Sprite()); // <-- HERE!!!

const graphics2 = createGraphics();
graphics2.position.set(200, 200);
app.stage.addChild(graphics2);

})();
```

### Environment

- **`pixi.js` version**: 8.0.0-rc.9
- **Browser & Version**: Chrome 121.0.6167.140
- **OS & Version**: Windows 11 22H2
- **Running Example**: <https://www.pixiplayground.com/#/edit/ZgzG_NMjOv4N588RvCbEz>

### Possible Solution

No idea at the moment. To be honest, I'm not very familiar with the new rendering system in PixiJS v8...

### Additional Information

The example code works just fine in Firefox 122 on my computer. Very strange...

**Comment by GoodBoyDigital** [comment]
interesting!! looks like the texture for the empty sprite is being used for the second graphics! Will investigate

**Comment by SuperSodaSea** [comment]
Fix confirmed in 8.0.0-rc.10. Thanks @GoodBoyDigital!
